drivers/gpu/drm/lima/lima_gem.c

```diff
@@ -426,3 +427,81 @@ int lima_gem_wait(struct drm_file *file, u32 handle, u32 op, u64 timeout_ns)
 	drm_gem_object_unreference_unlocked(obj);
 	return ret;
 }
+
+static struct lima_bo *lima_gem_alloc_object_dmabuf(struct drm_device *dev,
```

Comment:
Seems we can move this into drm_driver.gem_create_object which is just for the cma helpers like vc4 driver

Reply:
Another better method I think is to define a gem_prime_import_sg_table of lima to do the dma_buf import without create a gem cma obj. In this way we can also support none-continuous dma_buf import.